README.md

```diff
@@ -1,6 +1,7 @@
 # Project Nessie
 
 [![Build Status](https://github.com/projectnessie/nessie/workflows/Main%20CI/badge.svg)](https://github.com/projectnessie/nessie/actions)
+[![Java 17+18](https://github.com/projectnessie/nessie/actions/workflows/newer-java.yml/badge.svg)](https://github.com/projectnessie/nessie/actions/workflows/newer-java.yml)
```

Comment:
The badge renders as `Newer Java versions`, but as `Java 17+18` :thinking: Do we want to adjust that label?